Formula/jfrog-cli-go.rb

```diff
@@ -30,6 +30,6 @@ def install
   end
 
   test do
-    assert_equal "jfrog version 1.3.1", shell_output("#{bin}/jfrog -v").chomp
+    assert_equal "jfrog version 1.3.2", shell_output("#{bin}/jfrog -v").chomp
```

Comment:
`assert_equal "jfrog version #{version}"` will save this being updated every time.